README.md

```diff
@@ -1,6 +1,36 @@
 # patcher-action
-A GitHub Action for running Patcher
-
-This repo contains the example worflows for [Promotion Workflows with Terraform](https://blog.gruntwork.io/promotion-workflows-with-terraform-13c05bed953d) blog post.
+A GitHub Action for running Patcher.
 
 You can find out more about Gruntwork Patcher at [gruntwork.io](https://gruntwork.io/patcher) and by reading the [latest docs](https://docs.gruntwork.io/patcher/).
+
+## Usage
+
+Refer to the [/examples/github/workflows](/examples/github/workflows) folder for use cases of the action.
```

Comment:
@marinalimeira It seems like these examples do _not_ use the packaged JS Action in this PR... Shouldn't we update all those examples to do so?

Reply:
Yes! I will work on it today, it was the next step after making sure the workflows work in the dogfood repos.